src/foam/java/CodeProperty.js

```diff
@@ -23,6 +23,7 @@ foam.CLASS({
     {
       name: 'adapt',
       value: function(o, v) {
+        if ( v == 'false' || v == 'true') return v;
```

Comment:
What is this line for?

Reply:
if the code property is just a boolean, we need to output it rather than considering it as an entire instruction.